index.html

```diff
@@ -7,9 +7,27 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+ <header class="book-header"> 
+ <h1 class="book-title">Título del libro</h1>
+ <p class="book-genre">(género)</p>
+ </header>
+ <div class = "book-content">
+   <div class="book-content-row">
+      <h3 class="book-author-title">Autor del libro: </h3>
+      <p class="book-author-name">Nombre del autor del libro</p>
+   </div>
+   <div class="book-content-row">
+      <h3 class="book-editorial-title">Editorial: </h3>
```

Comment:
It is better to use margin to separate `h3` and `p` tag. And not this white space. Try without space and `margin-right`